setup.cfg

```diff
@@ -0,0 +1,56 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the 3-clause BSD style license (see LICENSE).
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+[metadata]
+name = apigentools
+version = 0.9.0.dev1
+description = Tooling for generation of API clients using openapi-generator
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords =
+    openapi
+    api
+    client
+    openapi-generator
+author = Slavek Kabrda
+author_email = slavek.kabrda@datadoghq.com
+url = https://github.com/DataDog/apigentools
+license = BSD 3-clause
+platforms = any
+classifiers=
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.7
+
+
+[options]
+packages = find:
+include_package_data = True
+scripts =
+    container-apigentools
```

Comment:
The `scripts` option (by design) does not work on Windows, just fyi

Reply:
yes, we should migrate it to a regular `console_scripts`.